server/manager/src/main/java/org/apache/accumulo/manager/upgrade/Upgrader9to10.java

```diff
@@ -105,12 +105,6 @@
  * <a href="https://github.com/apache/accumulo/issues/1642">#1642</a>, and
  * <a href="https://github.com/apache/accumulo/issues/1643">#1643</a> as well.</li>
  * </ul>
- *
- * Sorted recovery was updated to use RFiles instead of map files. So to prevent issues during
- * tablet recovery, remove the old temporary map files and resort using RFiles. This is done in
- * {@link #dropSortedMapWALFiles(ServerContext)}. For more information see the following issues:
- * <a href="https://github.com/apache/accumulo/issues/2117">#2117</a> and
- * <a href="https://github.com/apache/accumulo/issues/2179">#2179</a>
```

Comment:
This was probably fine to leave here, because it described the overall behavior of the Upgrader class.